xds/internal/balancer/clusterresolver/config_test.go

```diff
@@ -269,8 +261,16 @@ func TestParseConfig(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		b := balancer.Get(Name)
+		if b == nil {
+			t.Fatalf("LB policy %q not registered", Name)
+		}
+		cfgParser, ok := b.(balancer.ConfigParser)
+		if !ok {
+			t.Fatalf("LB policy %q does not support config parsing", Name)
+		}
```

Comment:
What do you think of this instead, parser := bb{};..... parser.ParseConfig() rather than this?

Reply:
I know we do such things at a lot of places. But I feel that even though my version is a little more verbose, it is using the exported API, rather than relying on internals. More often that not, I feel that in tests, this approach serves better in the long run.